hummingbird/ml/_container.py

```diff
@@ -82,8 +82,19 @@ def _fix_var_naming(operators, names, mod="input"):
 
     def forward(self, *inputs):
         with torch.no_grad():
-            assert len(self._input_names) == len(inputs)
-
+            assert len(self._input_names) == len(inputs) or (
+                hasattr(inputs[0], "columns") and len(self._input_names) == len(inputs[0].columns)
+            ), "numer of inputs or number of collumns in the dataframe do not match with the expected number of inputs {}".format(
+                self._input_names
+            )
+
+            if hasattr(inputs[0], "columns"):  # This is a pandas dataframe
```

Comment:
Is there a better way to check this?

Reply:
I can add a `pandas_installed()` check but I think it will have a hit in performance (since you load a library at runtime). Maybe I have a better idea, let me take a look.